### ISSUE TITLE
Word wrap

### ISSUE THREAD

**Issue by ainslec** [eclipse/tm4e#125]
Is there a way to enable word/line wrap in a TM4E editor?

I see that there is a way to enable this for Java editors ([here](https://stackoverflow.com/questions/2846002/does-eclipse-have-line-wrap)) but it would be HUGELY convenient to have a toggle on the context menu for TM4E editors.

The language that I'm working with often involves very very long lines of text, which are trivial to read with line wrapping and almost impossible to read without.

**Comment by angelozerr** [comment]
Eclipse Oxygen provides `Toggle Word Wrap` feature for any editor. 

![image](https://user-images.githubusercontent.com/1932211/28625339-3360b738-721c-11e7-8210-a1b22c6d3c74.png)

Please note that TME Editor is just for Neon. Oxygen provides GenericEditor and tm4e should use it https://github.com/eclipse/tm4e/issues/17

**Comment by ainslec** [comment]
Thanks for the toggle word wrap tip. It works very well.

I'm not sure what "TME Editor just being for Neon" means. In order to get syntax highlighting I had to go to preferences and set these options (import a .tmLanguage file, setup a theme):

![image](https://user-images.githubusercontent.com/201870/28644596-1ce35f16-7252-11e7-9a3a-af84f8254e41.png)

Then I had to go into file associations and associate my file types with the grammar..

![image](https://user-images.githubusercontent.com/201870/28644661-4e1ba502-7252-11e7-8cb8-9e091eeaca48.png)

If I try to open a file in the generic editor, it does not syntax highlight. And I am running Eclipse Oxygen. 

Or are you describing the plugin workflow perhaps?

Things are a little confusing between declaring all these things in a plugin, and also being able to configure them without a plugin in the Eclipse preferences. An FAQ might come in handy to explain the different approaches.
